pages/common/pyenv-virtualenv.md

```diff
@@ -10,3 +10,11 @@
 - List all existing virtual environments:
 
 `pyenv virtualenvs`
+
+- Activate a virtual environment:
+
+`pyenv activate {{my-virtual-env-3.6.6}}`
```

Comment:
Maybe `{{my-virtual-env-3.6.6}}` should be changed here, as well as the one in the first example.

Reply:
Agree, pinning a version in an example does not age well. May I suggest replacing `my-virtual-env-3.6.6` with `example`?

Reply:
Or even `virtualenv_name`.

Reply:
I agree with @ivanhercaz, `virtualenv_name` will do.

Reply:
Changed 👍